ext/RMagick/rmimage.c

```diff
@@ -12049,8 +12050,7 @@ Image_properties(VALUE self)
     Image *image;
     VALUE attr_hash;
     VALUE ary;
-    char *property;
-    const char *value;
+    const char *property, *value;
```

Comment:
Why do we put these on the same line, but not the `VALUE` lines above?

Reply:
Because I was changing property from a `char *` into a `const char *` and I did not notice that 😄. Thanks for noticing.